app/assets/stylesheets/layout/_profile.scss

```diff
@@ -75,69 +74,69 @@
   }
 }
 
-// .personal-footer {
-//  background: #fff;
-// }
-// .profile-header {
-//  background: #fdcc55;
-//  min-height: 248px;
-//  border-bottom: 20px $color-white solid;
-// }
-// .profile-wrapper {
-//  // @include make-row();
-//  color: $color-red;
+.personal-footer {
+ background: #fff;
+}
+.profile-header {
+ background: #fdcc55;
+ min-height: 248px;
+ border-bottom: 20px $color-white solid;
+}
+.profile-wrapper {
+ color: $color-red;
 
-//  .image-profile {
-//    // @include make-lg-column(3);
-//    position: relative;
-//    img {
-//      border: 10px solid $color-white;
-//      position: absolute;
-//      top: 58px;
-//      left: 20px;
-//      width: 260px;
-//      height: 260px;
-//      box-shadow: #000 3px 3px 23px;
-//    }
-//  }
-//  .profile-infos {
-//    // @include make-lg-column(9);
-//    padding-top: 48px;
-//    .profile-name {
-//      padding: 0;
-//      margin: 0;
-//      font-size: 56px;
-//      color: $color-red;
-//      font-weight: bold;
-//      float: left;
-//    }
-//    .edit-profile {
-//      float: right;
-//      color: $color-black;
-//      text-decoration: underline;
-//      font-size: 14px;
-//      margin-top: 20px;
-//    }
-//    .profile-description {
-//      display: block;
-//      float: left;
-//      color: $color-black;
-//      font-size: 15px;
-//      line-height: 19px;
-//      padding-top: 10px;
-//    }
-//  }
-// }
+ .image-profile {
+   // @include make-lg-column(3);
+   position: relative;
+   img {
+     border: 10px solid $color-white;
+     position: absolute;
+     top: 58px;
+     left: 20px;
+     width: 160px;
+     height: 160px;
+     box-shadow: #000 3px 3px 23px;
+   }
+ }
+ .profile-infos {
+   // @include make-lg-column(9);
+   padding-top: 48px;
+   .profile-name {
+     padding: 0;
```

Comment:
Properties should be ordered color, float, font-size, font-weight, margin, padding